x-pack/metricbeat/module/aws/terraform.tf

```diff
@@ -46,3 +46,21 @@ resource "aws_s3_bucket_object" "test" {
   bucket  = aws_s3_bucket.test.id
   content = "something"
 }
+
+resource "aws_instance" "test" {
+  ami           = data.aws_ami.latest-amzn.id
+  monitoring = true
+  instance_type = "t1.micro"
```

Comment:
Nit. Align the equal signs https://www.terraform.io/docs/language/syntax/style.html

Tip: configure your IDE to do it :slightly_smiling_face:

Reply:
😬 oh my it looks awful haha Thanks!!